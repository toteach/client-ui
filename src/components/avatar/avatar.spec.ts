import { mount, Wrapper } from '@vue/test-utils';
import Vue from 'vue';

import TAvatar from './avatar.vue';

describe('TAvatar', () => {
  let wrapper: Wrapper<Vue>;
  const image = 'https://www.fillmurray.com/64/64';
  beforeEach(() => {
    wrapper = mount(TAvatar, {
      propsData: {
        image,
      },
    });
  });
  it('is initialized', () => {
    expect(wrapper.name()).toBe('TAvatar');
    expect(wrapper.isVueInstance()).toBeTruthy();
  });
  it('renders correctly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
