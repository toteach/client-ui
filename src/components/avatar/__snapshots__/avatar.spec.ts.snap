// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TAvatar renders correctly 1`] = `"<figure class=\\"t-avatar\\"><img src=\\"https://www.fillmurray.com/64/64\\" alt=\\" \\" class=\\"t-avatar__image\\"></figure>"`;
